orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -54,6 +54,14 @@ base:
     - match: grain
     - postgresql
 
+  'roles:fluent_start':
+    - match: grain
+    - fluent.agent-start
+
+  'roles:fluent_stop':
```

Comment:
why do we need a stop role?

Reply:
as it's not really needed and agent-stop can be called manually, i can just simply use "fluent" on all nodes